bin/resources/GameIndex.yaml

```diff
@@ -20130,6 +20145,9 @@ SLES-55136:
 SLES-55138:
   name: "Nickelodeon El Tigre - The Adventures of Manny Rivera"
   region: "PAL-E"
+  gsHWFixes:
+    mipmap: 1
```

Comment:
missing comments for what it dose

Reply:
Okay, fixed. Also added comments to Barnyard highlighting remaining texture issues.